met-api/src/met_api/resources/report_setting.py

```diff
@@ -0,0 +1,97 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""API endpoints for managing report setting resource."""
+
+from http import HTTPStatus
+
+from flask import jsonify, request
+from flask_cors import cross_origin
+from flask_restx import Namespace, Resource
+from marshmallow import ValidationError
+
+from met_api.auth import auth
+from met_api.auth import jwt as _jwt
+from met_api.services.report_setting_service import ReportSettingService
+from met_api.utils.token_info import TokenInfo
+from met_api.utils.util import allowedorigins, cors_preflight
+
+
+API = Namespace('reportsetting', description='Endpoints for report setting management')
+"""Custom exception messages
+"""
+
+
+@cors_preflight('POST, OPTIONS, PATCH')
+@API.route('/')
+class ReportSetting(Resource):
+    """Resource for managing report setting."""
+
+    @staticmethod
+    # @TRACER.trace()
+    @cross_origin(origins=allowedorigins())
+    @_jwt.requires_auth
+    def post():
+        """Refresh the report setting to match the questions on survey."""
+        try:
+            requestjson = request.get_json()
+            user_id = TokenInfo.get_id()
+            ReportSettingService().refresh_report_setting(requestjson, user_id)
+
+            return {}, HTTPStatus.OK
```

Comment:
return the result of refresh_report_setting may be ..so that we can avoid etc GET call

Reply:
like how u did in patch